sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/PropagatorConfiguration.java

```diff
@@ -39,8 +39,11 @@ static ContextPropagators configurePropagators(
             serviceClassLoader);
 
     for (String propagatorName : requestedPropagators) {
-      propagators.add(
-          propagatorCustomizer.apply(getPropagator(propagatorName, spiPropagatorsManager), config));
+      if (!"none".equals(propagatorName)) {
```

Comment:
note: this means if someone does `none,baggage` it would work and just use the baggage propagator. That seems ok to me, since it's not really a valid configuration option, but it works as best we can guess it should.

Reply:
There are couple of other places where autoconfigure handles `none` and throws an exception if `none` is specified with another option. Examples here:
- [LogRecordExporterConfiguration](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/LogRecordExporterConfiguration.java#L44-L48)
- [SpanExporterConfiguration](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/SpanExporterConfiguration.java#L43-L47)
- [MeterProviderConfiguration](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MeterProviderConfiguration.java#L59-L63)

I think propagator configuration should behave the same.

Reply:
ok, updated to be less forgiving with bad config